requirements/dev-requirements.txt

```diff
@@ -2,7 +2,7 @@ pytest
 pytest-mock
 pytest-cov
 pep8
-astroid==1.4.8
-pylint==1.6.4
-tox==2.3.1
+astroid>=1.4.8
+pylint>=1.7.1
```

Comment:
This is dangerous because it will fail perfectly good build only because something new was added to pylint. I think that it is better to have some code style not checked rather than failing PR with good functionality to fail.